src/app/modules/group/components/manager-permission-dialog/manager-permission-dialog.component.html

```diff
@@ -48,11 +46,9 @@
           [collapsed]="false"
           (valueChange)="onValueChange()"
       >
-        <ng-template #label>
-          <span i18n>
-            User(s) can give and revoke members access to some content
-          </span>
-        </ng-template>
+        <span label i18n>
+          User(s) can give and revoke members access to some content
+        </span>
```

Comment:
boolean-section and progress-section should have the same API as before

Reply:
some important changes have been made, see [here](https://github.com/France-ioi/AlgoreaFrontend/pull/699#issuecomment-932177628)